### PR TITLE
TabPanel: Fix tabpanel container height (T1203780)

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js [hunk 1]
```diff
@@ -158,6 +159,25 @@ QUnit.module('rendering', {
...
+
+        const $tabPanel = $('<div>').appendTo('#qunit-fixture').dxTabPanel({
+            items,
+            width: 100,
+            height: 100,
+        });
+
+        const $tabPanelContainer = $tabPanel.find(`.${TABPANEL_CONTAINER_CLASS}`);
+
+        assert.strictEqual($tabPanelContainer.get(0).clientHeight, 62);
```
Comment (iBat) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js]: Why 62 if you set 100 via options?

Comment (EugeniyKiyashko) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js]: 'tabpanel-container' is element that wrap multiview. So, it's size exclude tabs container.

Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js]: dx-tabs height is 38px. dx-tabpanel-container is another element under dx-tabs, its height is 62px.

File: packages/devextreme/scss/widgets/base/_tabPanel.scss [hunk 2]
```diff
@@ -48,4 +48,5 @@
 
 .dx-tabpanel-container {
   flex-grow: 1;
+  overflow: hidden;
```
Comment (EugeniyKiyashko) [inline packages/devextreme/scss/widgets/base/_tabPanel.scss]: I suggest better way is using 'min-height: 0' for horizontal orientation, and 'min-width: 0' for vertical orientation. Also, the initial issue presented in our demo. I think we may improve our demo code.

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js]: I think we should test both of tabpanel configurations - vertical & horizontal.

File: packages/devextreme/scss/widgets/base/_tabPanel.scss (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/scss/widgets/base/_tabPanel.scss]: [318192470d093380e5908af4d01920d6e2ffd229]

File: packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js]: [318192470d093380e5908af4d01920d6e2ffd229]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by AlexanderMoiseev — APPROVED** [review]

**Pull request merged**